README.md

```diff
@@ -699,7 +700,7 @@ Input:
 - (Optional but recommended) `hostname` passed in query string
 - Request Message Body in json (`Content-Type` is `application/json`)
     - alias_prefix: string. The first part of the alias that user can choose.
-    - alias_suffix: should be one of the suffixes returned in the `GET /api/v2/alias/options` endpoint.
+    - signed_suffix: should be one of the suffixes returned in the `GET /api/v4/alias/options` endpoint.
```

Comment:
@ntnhon the alias creation is now using `signed-suffix` instead of just `suffix`. The current endpoint still works but is deprecated.